v0/destinations/tiktok_ads/data/TikTokTrack.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "destKey": "event_id",
+    "sourceKeys": "properties.eventId",
+    "required": false
+  },
+  {
+    "destKey": "timestamp",
+    "sourceKeys": "timestamp",
+    "required": false
+  },
+  {
+    "destKey": "test_event_code",
+    "sourceKeys": "properties.testEventCode",
+    "required": false
+  },
+  {
+    "destKey": "properties",
+    "sourceKeys": "properties.properties",
+    "required": false
+  },
+  {
+    "destKey": "context.ad",
+    "sourceKeys": ["context.ad", "properties.context.ad"],
+    "required": false
+  },
+  {
+    "destKey": "context.page",
+    "sourceKeys": ["context.page", "properties.context.page"],
+    "required": false
+  },
+  {
+    "destKey": "context.user",
+    "sourceKeys": ["context.user", "properties.context.user"],
+    "required": false
+  },
+  {
+    "destKey": "context.ip",
+    "sourceKeys": ["context.ip", "properties.context.ip"],
+    "required": false
+  },
+  {
+    "destKey": "context.user_agent",
+    "sourceKeys": ["context.userAgent", "properties.context.userAgent"],
+    "required": false
+  }
```

Comment:
You should prioritize the context field provided in properties. We don't have any user field in general inside the context field (you can refer to this https://www.rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/common-fields/#contextual-fields).

Reply:
Done